strings/sillytips.txt

```diff
@@ -1,3 +1,9 @@
 Clerks aren't meant to survive past 40 minutes. If they do - they are to be classified as an abnormality themselves for defying the laws of this place.
 We hire clerks as meat shields against abnormalities like Big Bird. Throw a clerk at it and attack the bird while it's chewing their head!
 Don't attack the crimson clowns that are operating the abnormality consoles, they are our system administrators performing maintenance check.
+You should pay your taxes before the Claw comes to vibe check you.
+Bongbriter isn't real. They can't hurt you.
+Do not simp the bees you sick freaks.
+WCCA has been updated with our latest safety system to avoid breaking bones during your knee cap surgery.
+Laetitia is a good girl who has done nothing wrong in her life.
+Stop teaching NT bad words.
```

Comment:
Claw is somewhat of a spoiler for new players(those that won't get to core suppression), idk how I feel about adding it to the tips.
"Bongbriter". So it is brittish?
WCCA tip feels obsolete and says nothing.